### PR TITLE
fix(react-tables): fix initialisation on tables on view, and copy and paste bugs

### PR DESCRIPTION
**Pull request by whawker**
### Description

Initialising editor content from HTML or Markdown, that included a React Table, did not initialise the table properly (tables controllers were never injected)

Additionally, copy and pasting a React table didn't check to see if controlled cells were already present. Leading them to be duplicated. (cc @ronnyroeller) 

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [ ] I have updated the documentation where necessary.
- [ ] New code is unit tested and all current tests pass when running `pnpm test`.

### CODE REVIEW

File: packages/remirror__pm/package.json [hunk 1]
```diff
@@ -185,6 +185,18 @@
   "publishConfig": {
     "access": "public"
   },
+  "@remirror": {
+    "tsconfigs": {
+      "src": {
+        "compilerOptions": {
+          "baseUrl": ".",
+          "paths": {
+            "prosemirror-tables": "node_modules/prosemirror-tables/src"
```
Comment (ocavue) [inline packages/remirror__pm/package.json:194]: Why do we need this? Is this related to the compilation error we saw before?

Comment (whawker) [reply packages/remirror__pm/package.json:194]: That was the theory. But looks like it doesn't work.

I was trying to avoid using the `prosemirror-tables/dist` which contains the error we're seeing
